core/logbook/src/main/resources/logbook_preferences.properties

```diff
@@ -0,0 +1,7 @@
+# ------------------------------
+# Package org.phoebus.logbook.ui
```

Comment:
I think this is typo, should say "org.phoebus.logbook" without .ui

Reply:
yes this was a typo...thanks for catching it